playground/src/main.tsx

```diff
@@ -51,7 +54,12 @@ function App(props: { katas: Kata[]; linkedCode?: string }) {
   const [compiler, setCompiler] = useState(() =>
     createCompiler(setCompilerState)
   );
-  const [evtTarget] = useState(new QscEventTarget(true));
+  const [evtTarget] = useState(() => new QscEventTarget(true));
+
+  const [languageService] = useState(() =>
+    getLanguageServiceWorker(languageServiceWorkerPath)
+  );
+  registerMonacoLanguageServiceProviders(languageService);
```

Comment:
Shouldn't this be called once in the useState that gets the language service to, rather than on every render?

Reply:
Darn, missed it. Sorry to make you have to review this one line at a time.